### PR TITLE
Fix: saving the form data

### PR DESCRIPTION
**Pull request by JonasVanHoof**
Everytime you go to the builder tab the refresh method is called with the `isInitialRouteCall` parameter on `true` this makes it that the `formChanged` variable is set to false as when initially lodaing the data there is no way that you already changed something in the form. 

I added a saved state of the variable `formChanged` to overwrite this but it feels that im hAcKinG more than we should @Windvis

### CODE REVIEW

**Comment by Windvis** [comment]
Any idea why the`isInitialRouteCall` needs to be set to true here in the first place?

**Comment by Windvis** [comment]
Anyway, it feels like we should prioritize https://github.com/lblod/frontend-form-builder/issues/17 😄 Things are hard to follow and it doesn't get easier with the extra form.

**Comment by JonasVanHoof** [comment]
> Any idea why the`isInitialRouteCall` needs to be set to true here in the first place?

This is done only to set these variable as when you initially load the form you could never have changed something in the builder (before it showed that the state is changed because it had already serialized the store graphs)
```js
    if (isInitialRouteCall) {
      this.setFormChanged(false);
      this.isAddingValidationToForm = false;
      this.isShowBuilder = true;
    }
